packages/framework/collabspaces/src/index.ts

```diff
@@ -11,5 +11,6 @@ export {
 	ICollabChannelCore,
 	ICollabChannel,
 	CollabSpaceCellType,
+	ReverseMapsDebugInfoType,
```

Comment:
Why? This is becoming public API. Tests do not require that - they can directly get to the guts of the implementation.

Reply:
Because it was used within  contract.ts - inside the IEfficientMatrixTest  which is also exported in  index.ts. 
Anyways, now that we split the implementation of getReverseMapsDebugInfo, I removed the type for now.